components/ui/data-table/data-table.tsx

```diff
@@ -68,29 +69,40 @@ export function DataTable<TData, TValue>({
     },
   });
 
+  const { generateClassnameForCell, generateClassnameForHeader } =
+    useDataTableStyles<TData, TValue>({ columns });
+
   const getAllVisibleData = () => {
```

Comment:
Any changes in `getAllVisibleData` is just my automatic formatter. We can undo these changes if you'd like, I have no preference. I'll explicitly flag the actual changes in this PR.